backend/sozisel/lib/sozisel/model/quizzes/track_node.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Sozisel.Model.Quizzes.TrackNode do
+  use Sozisel.Model.Schema
+  import Ecto.Changeset
+
+  @type t :: %__MODULE__{
+          fractional_second: Float.t(),
+          answer: String.t()
+        }
+
+  @primary_key false
+
+  embedded_schema do
+    field :fractional_second, :float
```

Comment:
I would name it `reaction_time` and add a description that is is a floaring point variable that means seconds and fraction of a second.